packages/flutter/lib/src/material/time_picker.dart

```diff
@@ -238,41 +234,73 @@ class _DayPeriodControl extends StatelessWidget {
     final TextStyle pmStyle = headerTextTheme.subhead.copyWith(
       color: !amSelected ? activeColor: inactiveColor
     );
-
-    return Column(
-      mainAxisSize: MainAxisSize.min,
-      children: <Widget>[
-        GestureDetector(
-          excludeFromSemantics: true,
-          onTap: Feedback.wrapForTap(() {
-            _setAm(context);
-          }, context),
-          behavior: HitTestBehavior.opaque,
-          child: Semantics(
-            selected: amSelected,
-            onTap: () {
-              _setAm(context);
-            },
-            child: Text(materialLocalizations.anteMeridiemAbbreviation, style: amStyle),
+    final bool layoutPortrait = orientation == Orientation.portrait;
+
+    final Widget amButton = ConstrainedBox(
+      constraints: _kMinTappableRegion,
+      child: Material(
+        type: MaterialType.transparency,
+        child: InkWell(
+          onTap: Feedback.wrapForTap(() => _setAm(context), context),
```

Comment:
The `=>` is usually reserved for one-liners that return a value. This is OK:
```
Feedback.wrapForTap(() { _setAm(context); }, context),
```

Here and below